diff -up kuser-16.08.3/ku_main.cpp.3~ kuser-16.08.3/ku_main.cpp
--- kuser-16.08.3/ku_main.cpp.3~	2020-09-03 00:42:46.384399833 +0200
+++ kuser-16.08.3/ku_main.cpp	2020-09-03 00:43:42.652722815 +0200
@@ -21,8 +21,9 @@
 
 #include "globals.h"
 
+#include <QApplication>
+
 #include <kiconloader.h>
-#include <kapplication.h>
 #include <kconfig.h>
 #include <klocale.h>
 #include <kcmdlineargs.h>
@@ -45,6 +46,7 @@ static const char *description =
 
 int main(int argc, char **argv) 
 {
+    QApplication a(argc, argv);
     K4AboutData aboutData("kuser", 0, ki18n("KUser"), _KU_VERSION, ki18n(description), K4AboutData::License_GPL,
                          ki18n("(c) 1997-2000, Denis Perchine\n(c) 2004, Szombathelyi György"), KLocalizedString(), 
                           "http://kde.org/applications/system/kuser/");
@@ -53,10 +55,9 @@ int main(int argc, char **argv)
     "dyp@perchine.com", "http://www.perchine.com/dyp/");
   aboutData.addAuthor(ki18n("Szombathelyi György"), ki18n("kuser author"),
     "gyurco@freemail.hu");
-  KCmdLineArgs::init(argc, argv, &aboutData);
+  KAboutData::setApplicationData(aboutData);
   KU_MainWidget *mw = 0;
 
-  KApplication a;
   KGlobal::locale()->insertCatalog( QLatin1String( "libkldap" ));
   KConfigGroup group( KGlobal::config(), "general" );
   KU_Global::initCfg( group.readEntry( "connection", "default" ) );
